anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc

```diff
@@ -38,82 +32,87 @@ When using the `prefetch` mode, the `acknowledgementTimeout` timer begins from t
 
 == Predictable Message Consumption
 
-If your app requires predictable and controlled message consumption rather than maximum throughput, you can configure `Subscriber` as a polling source. The Subscriber source checks for new messages in the queue at a fixed scheduling rate. Every time the poll is triggered, the Subscriber source retrieves up to 10 messages to be dispatched to the flow in order, one by one. To do so, configure `polling` in `subscriber-type`.
+If your app requires predictable and controlled message consumption rather than maximum throughput, you can configure the Subscriber as a polling source. The Subscriber source checks for new messages in the queue at a fixed scheduling rate. Every time the poll is triggered, the Subscriber source retrieves between 1 and 10 messages to dispatch to the flow individually in order. 
```

Comment:
The term presentation guide usually takes issue with capitalizing non-product names. Is it possible in this context to keep Subscriber lower case?

Reply:
I see later instances with Subscriber source, which would fit the standard, but reading this, I see why you've left it upper case.

Reply:
Subscriber is an operation - I just checked the SG and operations are init cap.

Reply:
Subscriber is an operation - I just checked the SG and operations are init cap.